docs/book/index.md

```diff
@@ -1,15 +1,17 @@
-## Installation
+# Installation
 
-### Using Composer
+## Using Composer
 
 ```bash
 $ composer require laminas/laminas-cache
 ```
 
 ## Learn
 
+```php
 <ul class="list-group list-group-flush">
     <li class="list-group-item">
         <a href="/laminas-cache/v3/application-integration/usage-in-a-laminas-mvc-application/">Usage in a laminas-mvc application</a>
     </li>
 </ul>
+```
```

Comment:
This is not a code example therefore this wrong.
See: https://docs.laminas.dev/laminas-cache/#learn

Use `<!-- markdownlint-disable no-inline-html -->` before the block and `<!-- markdownlint-enable no-inline-html -->` and after the block.

https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md033.md

Reply:
Fixed as per your recommendation